simpleSequential/integerToString.py

```diff
@@ -5,3 +5,5 @@
 #   integerToString(690) # return "690"
 #   integerToString(-69) # return "-69"
 #   integerToString(000) # return "0"
+def integerToString(n: int) -> int:
```

Comment:
I thought this is integer to string? The function header say something really different.

Reply:
The problem is fixed.